components/SubmitButton/SubmitButton.tsx

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+import React, { FunctionComponent } from 'react';
+
+interface BannerAsideProps {
```

Comment:
Rename appropriately, and make sure it extends all props of a button element.

Reply:
Renamed the interface and now extends HTMLButton element